.github/dependabot.yml

```diff
@@ -4,11 +4,14 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    open-pull-requests-limit: 10
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
       interval: "weekly"
+    open-pull-requests-limit: 10
```

Comment:
Realistically, we only need this limit for Maven, because Docker and GitHub actions never reach the default limit of 5, we only get a maximum of one or two each week for those two groups.